wavenet/model.py

```diff
@@ -84,8 +84,17 @@ def __init__(self,
         self.initial_filter_width = initial_filter_width
         self.histograms = histograms
 
+        self.receptive_field = self._calculate_receptive_field()
         self.variables = self._create_variables()
 
+    def _calculate_receptive_field(self):
+        receptive_field = (self.filter_width - 1) * sum(self.dilations) + 1
+        if self.scalar_input:
+            receptive_field += self.initial_filter_width - 1
+        else:
+            receptive_field += self.filter_width - 1
+        return receptive_field
```

Comment:
I don't think this is correct. I find that when:

```
dilations=[1,2,
           1,2]
scalar_input=false
filter_width=2
```
it yields receptive field = 8, whereas I think the correct value is 7:
![receptive_field](https://cloud.githubusercontent.com/assets/2138320/20745374/b4de6134-b6a6-11e6-9b60-9063d110bdbb.png)